Content.Shared/Alert/AlertsSystem.cs

```diff
@@ -2,13 +2,16 @@
 using System.Diagnostics.CodeAnalysis;
 using Robust.Shared.Player;
 using Robust.Shared.Prototypes;
+using Robust.Shared.Timing;
 
 namespace Content.Shared.Alert;
 
 public abstract class AlertsSystem : EntitySystem
 {
     [Dependency] private readonly IPrototypeManager _prototypeManager = default!;
 
+    private readonly IGameTiming _timing = IoCManager.Resolve<IGameTiming>();
```

Comment:
Don't resolve manually like this, it has to get the threadlocal collection every time which is slow compared to just using the dependency injection.

Reply:
Moved it to a dependency